vignettes/datatable-faq.Rmd

```diff
@@ -396,6 +396,7 @@ A key advantage of column vectors in R is that they are _ordered_, unlike SQL[^2
  - `check.names` is by default `TRUE` in `data.frame` but `FALSE` in data.table, for convenience.
  - `data.table` has always set `stringsAsFactors=FALSE` by default. In R 4.0.0 (Apr 2020), `data.frame`'s default was changed from `TRUE` to `FALSE` and there is no longer a difference in this regard; see [stringsAsFactors, Kurt Hornik, Feb 2020](https://developer.r-project.org/Blog/public/2020/02/16/stringsasfactors/).
  - Atomic vectors in `list` columns are collapsed when printed using `", "` in `data.frame`, but `","` in data.table with a trailing comma after the 6th item to avoid accidental printing of large embedded objects.
+ - Unlike data.frames a data.table cannot store rows with no columns: `nrow(DF[, 0])` returns the number of rows, while `nrow(DT[, 0])` always returns 0; but see issue [#2422](https://github.com/Rdatatable/data.table/issues/2422).
```

Comment:
I think this explanation misses to mention why it behaves like that. DF has simply matrix behaviour in that matter (discussed in various issues here and r devel mailing list as well). While DT way follows database table, rather than matrix, behaviour.

Reply:
@jangorecki I added a halve sentence concerning that